print/print-apps/oereb/toc.jrxml

```diff
@@ -301,13 +301,12 @@
 				</textElement>
 				<textFieldExpression><![CDATA[$R{FurtherInformation}]]></textFieldExpression>
 			</textField>
-			<textField hyperlinkType="Reference" hyperlinkTarget="Blank">
-				<reportElement x="260" y="105" width="232" height="11" forecolor="#4C8FBA" uuid="423aa9b7-82cf-47fe-8581-6ac646500ca7"/>
-				<textElement verticalAlignment="Top" markup="none">
-					<font fontName="Cadastra" size="6" isBold="true"/>
+			<textField isBlankWhenNull="true">
+				<reportElement x="260" y="105" width="232" height="11" uuid="423aa9b7-82cf-47fe-8581-6ac646500ca7"/>
+				<textElement verticalAlignment="Top" markup="html">
+					<font fontName="Cadastra" size="6"/>
 				</textElement>
-				<textFieldExpression><![CDATA["www.ne.ch/sgrf"]]></textFieldExpression>
-				<hyperlinkReferenceExpression><![CDATA["www.ne.ch/sgrf"]]></hyperlinkReferenceExpression>
+				<textFieldExpression><![CDATA["Weitere Informationen zum ÖREB-Kataster finden Sie auf unserer <a href=\"https://www.baselland.ch/politik-und-behorden/direktionen/volkswirtschafts-und-gesundheitsdirektion/amt-fur-geoinformation/oereb-kataster\"><font color=\"blue\">Webseite</font></a>"]]></textFieldExpression>
```

Comment:
Shouldn't be put in a print attribute as default value, or in the application config...

Reply:
Yes. See my comment above.

Reply:
It was hard coded until now. I want to change it to be read from Report.properties cause it is a cantonal but static content.

Reply:
@sbrunner: Can you make help out with reading the content of this field out of Report.properties?

Reply:
See sample PR :-)

Reply:
?

Reply:
https://github.com/camptocamp/pyramid_oereb/pull/403 :-)

Reply:
Ahhh :P